src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -255,8 +255,8 @@ def pretty_print(self):
     def export_to_csv(self, csv_filename: str):
         """Exports the statistics to a CSV file."""
 
-        header = [
-            "Statistic",
+        multiple_metric_header = [
```

Comment:
Might be a silly question, but can a common CSV library even read this format?

Reply:
I have the same question. From quick search over the internet, seems like it's possible but requires some workarounds on user's end.

Reply:
Matt and Hyunjae, could you clarify the ask here? Since this is an internal function that we'd be parsing, is the concern here that we'd have trouble parsing it or that external users need to use this for some reason and therefore need a commented example?

I think, ultimately, this will just output to a standard CSV. Though it could be nice to have parsing examples in our documentation. This would let us do the work once and make it easy for users going forward to use an example that shows how to parse out the information they want.

Reply:
I don't think there's any ask here. We were (or at least I was) just wondering if the multi-header csv format is a common/standard thing. 

Also is this an internal function? I thought this method is used to generate csv file for the genai-perf users to consume. Correct me if this is my misunderstanding.

Reply:
I see, I missed that you both were asking about generating two headers in the CSV and consuming the CSV.

Having an example in our documentation could be helpful. Something to note is that you could simplify parsing further by treating it as one header (let Pandas skip the empty line) and then just pull the value for the single-value metrics by calling "avg". The more correct way though would be to use something like Pandas to parse this as two tables.

Tim, if you have trouble with this documentation, let me know and I'm happy to help out.

Reply:
I'm thinking we should only be using the csv to transfer from PA->MA, and should use something more standard (like json) to output everything that someone consuming the script would want. Part of me wants to just add these outputs back into the profile-export.json file, so that all of the data that a user would want is in one place. That would be a different story though.